locales/fr-FR.json

```diff
@@ -0,0 +1,31 @@
+{
+	"An error occured: ": "Une erreur s'est produite. ",
+	"You have not linked your Spotify account, check your Alexa app to link the account": "Vous n'avez pas lié de compte Spotify, Vérifiez votre application Alexa afin de lier votre compte.",
+	"I can control your Spotify Connect devices, to start, ask me to list your devices": "Je peux contrôler vos appareils Spotify Connect, pour commencer, demandez-moi de lister vos appareils",
+	"To start, ask me to list your devices": "Pour commencer, demandez moi de lister vos appareils.",
+	"You can ask me to list your connect devices and then control them. ": "Vous pouvez me demander la liste de vos appareils connectés puis de les controler. ",
+	"For example, tell me to play on a device number after listing devices": "Par exemple, demandez moi de jouer sur le numéro d'un appareil après les avoir listés.",
+	"What would you like to do?": "Que voulez vous que je fasses ?",
+	"I couldn't work out the volume to use.": "Je n'ai pas pu calculer le volume à utiliser.",
+	"Try setting a volume between 0 and 10": "Essayez de régler le volume entre 0 et 10",
+	"You can only set the volume between 0 and 10": "Vous pouvez uniquement définir le volume entre 0 et 10",
+	"I couldn't find any connect devices, check your Alexa app for information on connecting a device": "Aucun appareil n'est connecté, vérifiez votre application Alexa pour plus d'informations sur la connexion d'un appareil",
+	"Connecting to a device using Spotify Connect": "Connexion à un appareils utilisant Spotify Connect",
+	"To add a device to Spotify Connect, log in to your Spotify account on a supported device such as an Echo, phone, or computer\nhttps://support.spotify.com/uk/article/spotify-connect/": "Pour ajouter un appareil à Spotify Connect, connectez-vous à votre compte Spotify sur un appareil compatible tel qu'un Echo, un téléphone ou un ordinateur \ nhttps: //support.spotify.com/fr/article/spotify-connect/",
```

Comment:
I noticed that there's an escaped space `\ ` where it looks like it should be `\n`, does it work as intended?

Reply:
If you would be able to fix that and push, and I'd be happy to merge after that

Reply:
It's done 👍